### PR TITLE
Add handling of page scope

### PR DESCRIPTION
**Pull request by solidgoldpig**
Increase coverage of code branching

Update dependencies

### CODE REVIEW

File: lib/load-json.js [hunk 1]
```diff
@@ -66,12 +66,10 @@ const getJsonSrc = sourceObj => {
...
     }))
   }
-  let jsonPaths
+  let jsonPaths = []
   try {
-    let testPath = fs.readdirSync(sourceObj.path)
-    if (testPath) {
-      jsonPaths = glob.sync(`${sourceObj.path}/**/*.json`)
-    }
+    fs.readdirSync(sourceObj.path)
```
Comment (umaar) [inline lib/load-json.js]: Without assigning it to a var, it feels a little side-effecty? Wonder if `fs.stat` would be more semantically correct? (in that here, we're not really reading a directory)

File: lib/load-json.js [hunk 2]
```diff
@@ -66,12 +66,10 @@ const getJsonSrc = sourceObj => {
...
+    jsonPaths = glob.sync(`${sourceObj.path}/**/*.json`)
```
Comment (umaar) [inline lib/load-json.js]: Is it correct that `glob.sync` is within the try block? Maybe not needed if the `readdirSync` is already being try/catched?

**Review by umaar — APPROVED** [review]
Minor comments but good to merge

File: lib/load-json.js (hunk 1 above)
Comment (solidgoldpig) [reply lib/load-json.js]: If we assign to a var 

this code is as is - the testPath condition was redundant

Semantically it's probably better as `fs.existsSync` which has been undeprecated. So I've switch to that

File: lib/load-json.js (hunk 2 above)
Comment (solidgoldpig) [reply lib/load-json.js]: `glob` doesn't throw if it doesn't find anything. But now the try/catch block has gone, this is moot

**Pull request merged**